Color/src/Graphics/Color/Space/OKLAB.hs

```diff
@@ -0,0 +1,158 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveTraversable #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeFamilies #-}
+-- |
+-- Module      : Graphics.Color.Space.OKLAB
+-- Copyright   : (c) Alexey Kuleshevich 2018-2020
+-- License     : BSD3
+-- Maintainer  : Alexey Kuleshevich <lehins@yandex.ru>
+-- Stability   : experimental
+-- Portability : non-portable
+--
+module Graphics.Color.Space.OKLAB
+  ( -- * Constructors for an CIE L*a*b* color space.
+    pattern OKLAB
+  , pattern ColorOKLAB
+  , pattern ColorOKLABA
+  , OKLAB
+  ) where
+
+import Foreign.Storable
+import Graphics.Color.Model.Internal
+import Graphics.Color.Space.Internal
+import Graphics.Color.Illuminant.ITU.Rec601 (D65)
+
+--------------
+--- OKLAB ---
+--------------
+
+-- | [OK L*a*b*](https://bottosson.github.io/posts/oklab/) color space
+data OKLAB
```

Comment:
```suggestion
-- 
-- Values for matrices used in conversion to and from @CIEXYZ@ where taken from [W3 CSS specification](https://www.w3.org/TR/css-color-4/#color-conversion-code) instead of the original blogpost about the OKLab color space.
data OKLAB
```